students/WesWang/lesson3/mailroom_part1.py

```diff
@@ -0,0 +1,86 @@
+#=========================================================================================
+#Name: Mailroom
+#Author: Wesley Wang
+#Date: 4/18/2018
+#=========================================================================================
+
+"""
+Mailroom Part1
+"""
+import sys
+
+donors = [["Donor A", 140000, 2],
```

Comment:
you want to store all the donations, not just the total and number. -- you can calculate the total and number of donations when you need them.